Source/lua/randomat2/events/gravity.lua

```diff
@@ -0,0 +1,77 @@
+local EVENT = {}
+
+CreateConVar("randomat_gravity_timer", 30, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Timer (def. 30)", 5,90)
+CreateConVar("randomat_gravity_duration", 3, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Duration (def. 3)", 1,15)
+CreateConVar("randomat_gravity_minimum", 70, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",10,500)
+CreateConVar("randomat_gravity_maximum", 2000, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",700,4000)
```

Comment:
Update the description to say "maximum" instead of "minimum"
Also, you can probably remove the "(def. ###)" parts since that isn't present in the rest of the events

Reply:
Guess I missed that one, all set